ownCloudAppShared/Client/Data Item Interactions/OCShare+Interactions.swift

```diff
@@ -148,9 +148,10 @@ extension OCShare: DataItemSelectionInteraction {
 					}
 				} else {
 					// Single share
-					editViewController = ShareViewController(mode: .edit, share: self, clientContext: context, completion: { _ in })
+					let item: OCItem? = try? context.core?.cachedItem(at: itemLocation)
```

Comment:
The changes in this file should not be needed, as the item is already fetched automatically in `ownCloudAppShared/Client/Sharing/ShareViewController.swift:126` if it is not passed to the `ShareViewController`.